.github/workflows/build-amd64.yml

```
@@ -134,11 +134,12 @@ jobs:
     - name: patch cmake implicit link libraries on macosx
       if: ${{ matrix.os == 'macosx' }}
       shell: bash
+      run: |
```

actionlint (1.7.4)

string should not be empty

(syntax-check)